src/rpc/blockchain.cpp

```diff
@@ -1203,6 +1208,7 @@ bool FillScRecordFromInfo(const uint256& scId, const CSidechain& info, CSidechai
 
             if (bVerbose)
             {
+                sc.push_back(Pair("certificateProvingSystem", static_cast<uint8_t>(info.fixedParams.certificateProvingSystem)));
```

Comment:
It would be better to add the prepending 'unconf' suffix as other attributes. The same for the analogous line below.